themes/classic/templates/catalog/_partials/product-images-modal.tpl

```diff
@@ -68,8 +67,11 @@
                         alt="{$product.name}"
                       {/if}
                       width="{$image.medium.width}"
+<<<<<<< HEAD
+=======
                       itemprop="image"
                       height="148"
+>>>>>>> fa6f2d0a1fcb02917fd789ee30b10cf848969ee0
```

Comment:
```suggestion
                      itemprop="image"
                      height="148"
```

Reply:
done

Reply:
without the itemprop (I removed all in my PR => image links are declared in JSONLD)